misc/labels.md

```diff
@@ -59,41 +59,41 @@ This issues is just for discussion or is concerning the build-process, the code-
 ## Additional labels
 
 ### `Duplicate`
-This issues is a duplicate of an existing issue. It should be closed naming the original issue it is a duplicate of.
+This issues is a duplicate of an existing issue. The duplicate should be closed. In addition, add a reference to the original issue with a comment.
 
 ### `Confirmed Bug`
-This issue was reported as `Problem` but the issue was reproducible and is now a confirmed bug.
+This issue was reported as `Problem`, but the issue is reproducible and is now a confirmed bug.
 
 ### `Work In Progress`
-Someone is working on this issue or an PR already exists and needs to be reviewed.<br>
+Someone is working on this issue or a pull request already exists and needs to be reviewed.<br>
 
 ### `Fixed awaiting release`
-This Issues was fixed or implemented in the "develop" branch to mark issues that have been fixed in development but are not released yet and therefore should be still open.<br>
-This issues should be closed if after the
+This Issues is fixed or implemented in the "develop" branch but is not released yet and therefore should be still open.<br>
+This issues should be closed if after the changes are merged into the "master" branch.
 
 ### `For everyone!`
 This is a good issue to start working on if you are new to vis.js and want to help.<br>
 This label is also used for labels that may concern a lot of vis.js users.
 
 ### `IE / Edge`
-This is issues concerns a problem in Microsoft Internet Explorer or the Edge browser.<br>
+These issues concern a problem with the Microsoft Internet Explorer or Edge browser.<br>
 
 ### `Issue Inactive`
-Issues marked as `question` or `problem` get marked as inactive if the author is not responsive or the topic is old.<br>
-If an issues is marked as inactive for about 2 weeks it can be closed without any hesitation.
+Issues marked as `question` or `problem` get marked as inactive when the author is not responsive or the topic is old.<br>
+If an issue is marked as inactive for about 2 weeks it can be closed without any hesitation.
 
 ### `PRIORITY`
-Normally this is used for major Bugs. There should only exist one or two issues marked as PRIORITY at the same time.<br>
-This issues need to be handled before all others.
+In general this is used for major bugs. There should only exist one or two issues marked as PRIORITY at the same time.<br>
+These issues need to be handled before all others.
 
 ### `Requires breaking change`
 A lot of code needs to be changed to implement this. This is maybe something for a major release or something for someone with a lot of time on their hands :-)
```

Comment:
Maybe we can merge this and discuss  in an seperate issue if we want to introduce 'Major', 'Minor', 'Patch' labels for Pull-Requests. (Instead of the old "Breaking change label).

@wimrijnders Can you open a new issue for that discussion please.